parsing/parser.mly

```diff
@@ -3780,8 +3765,9 @@ tuple_type:
 delimited_type_supporting_local_open:
   | LPAREN type_ = core_type RPAREN
       { type_ }
-  | LPAREN MODULE attrs = ext_attributes package_type = package_type RPAREN
-      { wrap_typ_attrs ~loc:$sloc (reloc_typ ~loc:$sloc package_type) attrs }
+  | LPAREN MODULE ext_attrs = ext_attributes package_type = package_type RPAREN
+      { mktyp_attrs ~loc:$sloc
+          (reloc_typ ~loc:$sloc package_type).ptyp_desc ext_attrs }
```

Comment:
Note for later: this is correct because the type from `package_type` doesn't have attributes currently (the attributes are now stored inside the `Package_type` record themselves.

Reply:
Done.